webtau-rest-groovy/src/test/groovy/com/twosigma/webtau/http/HttpTest.groovy

```diff
@@ -313,6 +321,28 @@ class HttpTest {
         http.doc.capture("end-point-dates-matchers")
     }
 
+    @Test
+    void "matchers combo"() {
+        http.get("/end-point-mixed") {
+            list.should contain(lessThanOrEqual(2)) // lessThanOrEqual will be matched against each value
```

Comment:
Just to make sure I understand correctly, this is conceptually equivalent to:
```
list.stream().any(x -> x <=2)
```
rather than `all()`?

Reply:
yes. should the function names be changed to make it less confusing?

Reply:
It's not that it was confusing.  The actual behaviour is what I would have inferred from the names, just wanted to make sure it definitely wasn't the other one.

Only alternative I can think of would be for the lessThanOrEqual and friends which are intended to be used in composition should be anElementLessThanOrEqual?  But that'll only make sense when composed with a contain not something else...